shared-images/airflow/dist/openshift_plugin/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+graft openshift_plugin/templates
+graft openshift_plugin/static
```

Comment:
Because it is my implementation, there is not something like OpenShift plugin for AirFlow... It should became a package or a direct contribution to Airflow, but we are not there yet

Reply:
Aha :) Then forget my comment!